{
	"id": "CVE-2023-46377",
	"sourceIdentifier": "cve@mitre.org",
	"vulnStatus": "Rejected",
	"published": "2023-11-21T16:15:42.713",
	"lastModified": "2023-11-21T16:15:42.713",
	"descriptions": [
		{
			"lang": "en",
			"value": "Rejected reason: DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none."
		}
	],
	"references": [],
	"metrics": {}
}
